stix_shifter/stix_translation/src/modules/bigfix/json/from_stix_map.json

```diff
@@ -0,0 +1,49 @@
+{
+  "file": {
+    "fields": {
+      "name": ["file.name"],
+      "hashes.SHA-256": ["file.sha256"],
+      "hashes.SHA-1": ["file.sha1"],
+      "hashes.md5": ["file.md5"],
```

Comment:
The STIX md5 attribute should be uppercase here. Previous hash mappings in STIX-shifter have followed the example in the STIX documentation as seen in https://docs.google.com/document/d/1nK1RXcE2aMvQoG1Kgr3aTBtHZ1IyehzOk7vU0n5FUGY/edit#heading=h.jphmvccraytb.

Reply:
Incorporated with latest Commit.